### PR TITLE
Annoyed Alligators

### PR DESCRIPTION
**Pull request by akshgpt7**
Annoyed Alligators' PR

1. You'll need to have `EMAIL_USER` and `EMAIL_PASSWORD` environment variables stored in a .env file for the password reset to work.
2. flake8 errors are only inside the standard django generated files which went unused.

### CODE REVIEW

**Comment by SebastiaanZ** [comment]
One commit, a one-line typo fix in their README, was made about 39 minutes after the deadline.

---
flake8 linting errors

```
./annoyed-alligators/socl_media/apps/chat/urls.py:1:1: F401 'django.urls.path' imported but unused
./annoyed-alligators/socl_media/apps/chat/tests.py:1:1: F401 'django.test.TestCase' imported but unused
./annoyed-alligators/socl_media/apps/feed/tests.py:1:1: F401 'django.test.TestCase' imported but unused
./annoyed-alligators/socl_media/apps/terminal/admin.py:1:1: F401 'django.contrib.admin' imported but unused
./annoyed-alligators/socl_media/apps/terminal/tests.py:1:1: F401 'django.test.TestCase' imported but unused
./annoyed-alligators/socl_media/apps/users/urls.py:1:1: F401 'django.urls.path' imported but unused
./annoyed-alligators/socl_media/apps/users/views.py:7:1: F401 '.models.Profile' imported but unused
./annoyed-alligators/socl_media/apps/users/tests.py:1:1: F401 'django.test.TestCase' imported but unused
./annoyed-alligators/socl_media/apps/users/apps.py:8:9: F401 'socl_media.apps.users.signals' imported but unused
```

File: annoyed-alligators/README.md [hunk 1]
```diff
@@ -1,2 +1,163 @@
-# Annoyed Alligators
-This is the default README of your team's project. Please replace this by a README with more information on your project. At the very least, your README should contain information on how to set-up and run your project.
+<div align="center">
```
Comment (eivl) [inline annoyed-alligators/README.md:1]: Well made README.md

File: annoyed-alligators/socl_media/apps/feed/models.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    def get_absolute_url(self):
+        return reverse('post-detail', kwargs={'pk': self.pk})
+
+    def save(self, **kwargs):
+        """
+        This function will resize and add a pixelating effect to the images
+        to give them a more retro and early internet feel.
+        """
+        super().save()
```
Comment (eivl) [inline annoyed-alligators/socl_media/apps/feed/models.py:31]: First things that comes to mind is that *args and **kwargs might be useful if you ever want to use `super`

File: annoyed-alligators/socl_media/apps/terminal/commands.py [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+
+    return {'response': add_linebreaks(message)}
+
+
+message = Command(name='message', help_text="""<br>Send a personal message to any user.<br>
+Usage: message [username] [message body]""",
+                  action=message_func)
+
+
+def message_box_func(request, params: str, **kwargs):
```
Comment (eivl) [inline annoyed-alligators/socl_media/apps/terminal/commands.py:244]: I like short functions like this one. It makes it easy to follow, and clear.

File: annoyed-alligators/socl_media/apps/terminal/commands.py [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+    return {'response': add_linebreaks(message), 'redirect': redirect}
+
+
+profile = Command(name='profile', help_text="""<br>View any user's profile.<br>
+You'll see your own profile if no username is specified.<br>
+Usage: profile [username]""",
+                  action=profile_func)
+
+
+def news_func(request, topic: str, start_date: str = None, end_date: str = None, **kwargs):
```
Comment (eivl) [inline annoyed-alligators/socl_media/apps/terminal/commands.py:293]: Looks like there are a lot of things going on here. I find it a bit hard to understand.
but that said, nice use of list-comprehension.

File: annoyed-alligators/socl_media/apps/terminal/models.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    followup response.
+    """
+
+    user_id = models.IntegerField()
+    search_topic = models.CharField(max_length=50)
+    search_time = models.DateTimeField(auto_now=True, auto_now_add=False)
+    last_fetched_count = models.IntegerField()
+    news_articles = models.TextField()
+
+    def __str__(self):
```
Comment (eivl) [inline annoyed-alligators/socl_media/apps/terminal/models.py:16]: nice use of `__str__`
not sure if its ever used, but still nice

File: annoyed-alligators/socl_media/apps/users/views.py [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+    def get_context_data(self, *args, **kwargs):
+        context = super(profile, self).get_context_data(*args, **kwargs)
+        context['of_user'] = self.get_of_user()
+        context['num_posts'] = len(self.user_posts)
+        return context
+
+
+@login_required
+def profile_edit(request):
```
Comment (eivl) [inline annoyed-alligators/socl_media/apps/users/views.py:60]: nice to see some docstrings. they really help out.

**Pull request merged**